src/plugins/links/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -189,6 +189,7 @@ export const DashboardLinkComponent = ({
       label={linkLabel}
       external={link.options?.openInNewTab}
       data-test-subj={error ? `${id}--error` : `${id}`}
+      aria-current={link.destination === parentDashboardId}
```

Comment:
I had to look this one up to make sure `aria-current="false"` was valid, and turns out yes, that's perfectly OK:

> The `aria-current` attribute accepts a limited list of [values](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#values) including `page, step, location, date, time, true,` and `false`. Any non-null string value not included in this list of enumerated values is treated as if `aria-current="true"` were set, not the default `false` value. If the attribute is not present, is an empty string, is present with no value, or is set to `aria-current="false"` it is **not exposed** to the user.
>
>https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current